observability/ossm-observability-service-mesh-assembly.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ossm-observability-service-mesh-assembly"]
+= Observability and Service Mesh
+include::_attributes/common-attributes.adoc[]
+:context: ossm-observability-assembly
+
+toc::[]
+
+Red Hat OpenShift Observability provides real-time visibility, monitoring, and analysis of various system metrics, logs, and events to help you quickly diagnose and troubleshoot issues before they impact systems or applications.
```

Comment:
Should this be Red{nbsp}Hat throughout the file?
```suggestion
Red{nbsp}Hat OpenShift Observability provides real-time visibility, monitoring, and analysis of various system metrics, logs, and events to help you quickly diagnose and troubleshoot issues before they impact systems or applications.
```

Reply:
For my own edification: Does Red{nbsp}Hat only apply if you aren't using attributes?

https://github.com/openshift/openshift-docs/blob/main/_attributes/common-attributes.adoc 

none of them use Red{nbsp}Hat, so curious

Reply:
(I had to hunt this up; I just saw another reviewer mention the nbsp in another PR.)
@gwynnemonahan From the [Supplementary Style Guide](https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces): Use a non-breaking space ({nbsp}) between the words "Red" and "Hat". 
By this, it seems that the attributes should use the {nbsp}.